anaconda.spec.in

```diff
@@ -144,7 +144,7 @@ Requires: python3-coverage >= 4.0-0.12.b3
 Requires: anaconda-tui = %{version}-%{release}
 
 # Make sure we get the en locale one way or another
-Requires: (glibc-langpack-en or glibc-all-langpacks)
+Requires: glibc-langpack-en
```

Comment:
I don't really see how this should end up installing both ``glibc-langpack-en`` and ``glibc-all-langpacks``. Or is the latter installed by lorax?